package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "start": "webpack-dev-server --mode development --open",
-    "build": "webpack --mode production",
+    "build": "webpack --mode production --progress",
```

Comment:
why --progress means?

Reply:
* Adds a progress bar to the build process